apps/design/backend/src/language_and_audio/ballot_strings.ts

```diff
@@ -29,5 +30,10 @@ export async function translateBallotStrings(
     machineVersion,
     ballotLanguageConfigs
   );
-  return mergeUiStrings(electionStrings, appStrings);
+  // Temporary hack: Only pass the HMPB app strings to the renderer.
+  // TODO: construct and translate these as a separate package
```

Comment:
In my next PR, I'll be building a nicer solution for HMPB app strings, likely a separate UI strings module rather than including them in `appStrings`